runtime/src/main/java/io/quarkiverse/bucket4j/runtime/RateLimiterRuntimeConfig.java

```diff
@@ -32,4 +36,55 @@ public interface RateLimiterRuntimeConfig {
     @WithConverter(DurationConverter.class)
     Duration keepAfterRefill();
 
+    /**
+     * represent a group of limit applied to a method
+     * identified by the bucket id
+     */
+    @ConfigDocMapKey("bucket-id")
+    Map<String, Bucket> buckets();
```

Comment:
Take a look at how I did in https://github.com/quarkiverse/quarkus-ironjacamar/blob/main/runtime/src/main/java/io/quarkiverse/ironjacamar/runtime/IronJacamarRuntimeConfig.java#L37, maybe it could use something like that here?